model/qti/templates/feedbacks/rules/qti.choices.tpl.php

```diff
@@ -42,5 +42,5 @@
         <setOutcomeValue identifier="<?=$feedbackOutcomeIdentifier?>">
     <baseValue baseType="identifier"><?=$feedbackIdentifierElse?></baseValue>
             </setOutcomeValue>
-<?php endif;?>
+    </responseElse><?php endif;?>
```

Comment:
Could you please align the opening tag on line 41, so it's more obvious what's going on there?